packages/cli/src/databases/db-connection.ts

```diff
@@ -80,9 +84,12 @@ export class DbConnection {
 		}
 	}
 
-	/** Ping DB connection every 2 seconds */
+	/** Ping DB connection every `pingIntervalSeconds` seconds to check if it is still alive. */
 	private scheduleNextPing() {
-		this.pingTimer = setTimeout(async () => await this.ping(), 2000);
+		this.pingTimer = setTimeout(
+			async () => await this.ping(),
+			this.databaseConfig.pingIntervalSeconds * Time.seconds.toMilliseconds,
```

Comment:
we could really benefit from a `Duration` type to make all the timeouts a tiny bit more type-safe.